### DISCUSSION TITLE
Famicom adapter PCB

### DISCUSSION THREAD

**Discussion by Just001Kim** [sanni/cartreader#70] [Need help]
Hi,

I received a email from jlcpcb about the famicom to snes adapter.
It seems that the template is not fully opened, can it be that this is correct?  I’l add what I recieved in pictures.

![69678532-0AE8-49D8-B3EF-6C77049D99AE](https://user-images.githubusercontent.com/61664634/104172618-da0fc100-5404-11eb-8046-7484014486b4.jpeg)
![8DE939EB-14B2-41E2-95A6-AFA8A2550D0B](https://user-images.githubusercontent.com/61664634/104172623-dbd98480-5404-11eb-88c8-9107117eb9e5.png)
![F089532F-1B14-41EE-9747-7AF89120AD02](https://user-images.githubusercontent.com/61664634/104172626-dc721b00-5404-11eb-865d-0d3d0163071d.png)

**Comment by sanni** [comment]
Yes "not fully opened" is correct, original Nintendo carts have the same thing to prevent corrosion on the small traces above.

![SNS-ATFE-0-pcb-front-94xx](https://user-images.githubusercontent.com/19169494/104177444-0e857c00-5409-11eb-8b70-b1a5194c9d8d.jpg)

**Reply by Just001Kim** [reply]
Ok, didn’t know that.
Thanks for the reply and the project!

**Comment by sanni** [comment]
EA on the other hand did not do that with their Mega Drive games and as you can see in this picture the small traces are exposed and therefore can easily corrode which would result in dead game.

![MtNIF](https://user-images.githubusercontent.com/19169494/104177992-f104e200-5409-11eb-8dda-28639f2741e1.jpg)
